qcfractal/config.py

```diff
@@ -100,6 +102,9 @@ class Config(SettingsCommonConfig):
 
 class FractalConfig(ConfigSettings):
 
+    # class variable, not in the pydantic model
+    defaults_file_path: ClassVar[str] = os.path.expanduser("~/.qca/qcfractal_defaults.yaml")
```

Comment:
The other thing to do here is use a `_defaults_file_path`. This will tell pydantic to automatically consider it a class var.